File: 3019.number-of-changing-keys.py
#
# @lc app=leetcode id=3019 lang=python3
#
# [3019] Number of Changing Keys
#

# @lc code=start
class Solution:
    def countKeyChanges(self, s: str) -> int:
        result, n = 0, len(s)
        for i in range(n - 1):
            if s[i + 1].lower() != s[i].lower(): result += 1
        return result
# @lc code=end

File: 3019.number-of-changing-keys.cs
/*
 * @lc app=leetcode id=3019 lang=csharp
 *
 * [3019] Number of Changing Keys
 */

// @lc code=start
public class Solution {
    public int CountKeyChanges(string s) {
        int result = 0;
        int n = s.Length;
        for (int i = 0; i < n - 1; i++) {
            if (char.ToLower(s[i + 1]) != char.ToLower(s[i])) result++;
        }
        return result;
    }
}

// @lc code=end

